manifests/capacityscheduling/scheduler-config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kubescheduler.config.k8s.io/v1alpha2
+kind: KubeSchedulerConfiguration
+leaderElection:
+  leaderElect: false
+clientConnection:
+  kubeconfig: /etc/kubernetes/framework-config.yaml
+profiles:
+  - schedulerName: default-scheduler
+    plugins:
+      preFilter:
+        enabled:
+          - name: CapacityScheduling
+      postFilter:
+        enabled:
+          - name: CapacityScheduling
```

Comment:
Need to disable the default preemption plugin.

Reply:
Done thanks ~